packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcStop.test.ts

```diff
@@ -32,4 +61,26 @@ describe('forceLightningLwcStop', () => {
     await forceLightningLwcStop();
     sinon.assert.calledOnce(stopStub);
   });
+
+  it('shows successful server stop', async () => {
```

Comment:
Also added a test for the Stop command to ensure it prints the successful stop text during this scenario.